src/features/student/declaredPrograms/components/interactions/inputs/DeclaredProgramDescriptionTextarea/DeclaredProgramDescriptionTextarea.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import { DECLARED_PROGRAM_DESCRIPTION_MAX_LENGTH } from '@/features/student/declaredPrograms/config'
+import { AvInput, type AvInputProps } from '@avenirs-esr/avenirs-dsav'
+import { useI18n } from 'vue-i18n'
+
+type DeclaredProgramDescriptionTextareaProps = Omit<AvInputProps, 'maxlength' | 'isTextarea'>
+
+const {
+  label,
+  ...restProps
+} = defineProps<DeclaredProgramDescriptionTextareaProps>()
+
+const modelValue = defineModel<string>()
+const { t } = useI18n()
+
+const avInputProps = computed(() => ({
+  ...restProps,
+  isTextarea: true,
+  labelVisible: true,
+  maxlength: DECLARED_PROGRAM_DESCRIPTION_MAX_LENGTH,
+  label: label ?? t('student.declaredPrograms.interactions.inputs.DeclaredProgramDescriptionTextarea.label')
+}))
+</script>
+
+<template>
+  <div class="declared-program-description-textarea">
+    <AvInput
+      v-bind="avInputProps"
+      v-model="modelValue"
+    >
+      <template
+        v-if="!$slots.customCaptions"
+        #customCaptions="{ currentValue }"
+      >
+        <span class="caption-light">
+          {{ t('global.inputs.textarea.limit', {
+            count: currentValue?.toString().length || 0,
+            maxlength: avInputProps.maxlength,
+          }) }}
+        </span>
+      </template>
```

Comment:
```suggestion
      <template #customCaptions="{ currentValue }">
        <slot name="customCaptions">
          <span class="caption-light">
            {{ t('global.inputs.textarea.limit', {
              count: currentValue?.toString().length || 0,
              maxlength: avInputProps.maxlength,
            }) }}
          </span>
        </slot>
      </template>
```

Ça fonctionne pas ça ?

Reply:
(y)
en vrai pas besoin de check ni permettre de custom le slot vu que c'est tjr le meme